docs/translate.md

```diff
@@ -58,8 +58,8 @@ You will be prompted with a dialog that contains the block translatable string.
 
 ### Documentation page translation
 
-In any documention page, click on the **Language** button 
-in the footer; then select translation this page in the dialog.
+When on any documention page, click on the **Language** button 
```

Comment:
there seems to be a problem with this feature. it works on localhost but not in the cloud. The current way to do this is to add ?translate=1 on the page

Reply:
Add a note for that?

Reply:
let's see if it's a quick fix first